### ISSUE TITLE
Some questions about training  params and times

### ISSUE THREAD

**Issue by SlashChven** [HIT-SCIR/ELMoForManyLangs#10]
Hi, i am trying to train my own elmo by using your train script, but it cost much time.
I notice that "The training of ELMo on one language takes roughly 3 days on an NVIDIA P100 GPU."
Can you tell me the size of one language corpus?  The whole training progress means 100 epochs?

Also, can you share you params? Such as learning rate, learning rate decay, batch_size, optimizer.

Thanks

**Comment by Oneplus** [comment]
The training data for each language consist about 20M words. We train it for 10 epochs.

For the training parameters, you can refer the `${lang}.model/config.json`.
